WcaOnRails/app/models/round.rb

```diff
@@ -80,7 +80,7 @@ def self.wcif_to_round_attributes(wcif, round_number)
   end
 
   def wcif_id
```

Comment:
No gripes with this commit.

Just a comment: This commit is to fix formatting this "wcif_id" to match the activity codes, do we want to call this an activity code so we can start using activity codes, or just leave it as a "wcif_id".

Reply:
Oh, that might be better!

However, I'm going to leave this as is for now, because we also have `self.parse_wcif_id` and `self.wcif_id_to_name` methods that would have to be changed for consistency, which would then affect other code that uses those methods, and I want to keep this PR simple.

Maybe it would make sense to remove the id attribute from Round in the WCIF and rename it to `activity_code`?